src/sentry/static/sentry/app/views/organizationDiscover/result/index.jsx

```diff
@@ -193,6 +193,7 @@ export default class Result extends React.Component {
                 height={300}
                 tooltip={tooltipOptions}
                 legend={{data: [baseQuery.query.aggregations[0][2]], truncate: 80}}
+                xAxis={{truncateXAxis: true}}
```

Comment:
I would rename this to just `truncate`, since it's implied that it's for the xaxis. Also instead of having it boolean, it should be an int